api/src/test/java/org/openmrs/module/ModuleFileParserTest.java

```diff
@@ -71,17 +69,15 @@ public static void setUp() throws ParserConfigurationException {
 	@Test
 	public void moduleFileParser_shouldFailCreatingParserFromFileIfGivenNull() {
 
-		expectModuleExceptionWithTranslatedMessage("Module.error.fileCannotBeNull");
-
-		new ModuleFileParser((File) null);
+		ModuleException exception = assertThrows(ModuleException.class, () -> new ModuleFileParser((File) null));
+		assertThat(exception.getMessage(), is(messageSourceService.getMessage("Module.error.fileCannotBeNull")));
 	}
 
 	@Test
 	public void moduleFileParser_shouldFailCreatingParserFromFileIfNotEndingInOmod() {
 
-		expectModuleExceptionWithTranslatedMessage("Module.error.invalidFileExtension");
-
-		new ModuleFileParser(new File("reporting.jar"));
+		ModuleException exception = assertThrows(ModuleException.class, () -> new ModuleFileParser(new File("reporting.jar")));
+		assertThat(exception.getMessage(), is("Module file does not have the correct '.omod' file extension Module: reporting.jar"));
```

Comment:
```suggestion
		assertThat(exception.getMessage(), startsWith(messageSourceService.getMessage("Module.error.invalidFileExtension")));
```

If startsWith is not working use containsString